app/components/Map/map_component.jsx

```diff
@@ -7,89 +7,146 @@ import InputLabel from '@material-ui/core/InputLabel';
+                {address: "East Oakland Collective", mile: "0.7 mi", hour: "2 hrs"},
+                {address: "Oakland YMCA", mile: "1 mi", hour: "3 hrs"},
+                {address: "Tri-City Health Center", mile: "1.5 mi", hour: "2 hrs"}
+            ]
+        }
+
+        this.resource = false;
+
+         this.handleClick = this.handleClick.bind(this)
+    }
+
+    handleClick(){
+
+        // Changing state
+        //this.setState({resource:true})
+
+        this.resource =true;
+    }
+    handleClickAway = () => {
+        //This is not being called when clicking on the button below
+        setClickAwayStatus('Registered onClickAway event from ClickAwayListener');
+    };
+
+
+
+
+    render() {
+        var {navigation} = this.props;
+        console.log(this.resource)
+
+        return (
+
+
+            <View >
+                <HeaderComponent style={mapStyles.main}
+                                 navigation={navigation}
+                                 title='Map'
                 >
-                    <MenuItem > Mental Health </MenuItem>
-                    <MenuItem > LGBTQ </MenuItem>
-                    <MenuItem > BLM </MenuItem>
-                    <MenuItem > Food Banks </MenuItem>
-                    <MenuItem > Health Centers </MenuItem>
-                    <MenuItem > Community cleanup/art </MenuItem>
-
-
-                </Select>
-            </DropDownMenuProps>
-
-
-        </Grid>
-
-        {(address).map((address, )=> (
-            <Card>
-                {/*<Card.Title title="Card Title" subtitle="Card Subtitle" left={LeftContent} />*/}
-                <Card.Content >
-                    <Title class = "card">
-                        {address.address}
-                        {"      "}
-                        {"Hours"}
-                    </Title>
-                    <Paragraph>
-                        {address.mile}
-                        {"      "}
-                        {address.hour}
-                    </Paragraph>
-                </Card.Content>
-                {/*<Card.Cover source={{ uri: 'https://picsum.photos/700' }} />*/}
-
-            </Card>
-
-        ))}
-
-      <ToolbarComponent style={mapStyles.navigation} navigation={navigation}></ToolbarComponent>
-    </View>
-  )
-}
-
-export default MapComponent
\ No newline at end of file
+                </HeaderComponent>
+
+                <Button onClick={this.handleClick()}>map</Button>
+
+
+                {(() => {
+                    switch (this.resource) {
+                        case (false):
+                            return <header>false</header>
+                        case (true):
+                            return <header>true</header>
+
+
+                        default:
+                            return (
+                                <header>true</header>
+                        );
+                    }
+                })()}
+
+
+
+                {/*<Grid container alignItems="center">*/}
```

Comment:
Remove code (files can always be reverted :)) or add a comment explaining the change.